### DISCUSSION TITLE
New Config/Palette Thread

### DISCUSSION THREAD

**Discussion by djyt** [djyt/system16_sprite_viewer#2] [Show and tell]
This is a thread to list additions config & palette additions.

1/ I added Shinobi. No sign of the alternate palettes for the hands shown in the [flyer](https://flyers.arcade-museum.com/?page=flyer&db=videodb&id=6345&image=1). 

![shinobi1](https://user-images.githubusercontent.com/2414449/115965005-05836b00-a51f-11eb-9807-7bb012b9465e.png)

2/ CMonkey added support for WonderBoy 3.

![wb3_title](https://user-images.githubusercontent.com/2414449/115965034-1cc25880-a51f-11eb-942f-70b41656a10d.jpeg)

**Comment by dlfrsilver** [comment]
Hello djyt, can you compile your tool Sega S16 Viewer for Windows 32 bits ? My main machine is still 32 bits :/ 
Thanks !
